docs/tasks/administer-cluster/ip-masq-agent.md

```diff
@@ -0,0 +1,97 @@
+---
+title: IP Masquerade Agent User Guide
+---
+
+{% capture overview %}
+This page show how to configure and enable the ip-masq-agent. 
+{% endcapture %}
+
+{% capture prerequisites %}
+Kubernetes 1.7
+{% endcapture %}
+
+{% capture discussion %}
+## IP Masquerade Agent User Guide
+
+The ip-masq-agent configures iptables rules to hide a pod's IP address behind the cluster nodes's IP address. This is typically done when sending traffic to destinations outside the cluster's pod [CIDR](https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing) range.
+
+## **Key Terms**
```

Comment:
This should probably be a level 3 heading since it is part of the User Guide.

`### **Key Terms**`